lib/cob_index/suppress_config.rb

```diff
@@ -7,16 +7,14 @@
 
 settings(&CobIndex::DefaultConfig.indexer_settings)
 
-
-to_field "status", extract_status
-
 each_record do |record, context|
-
-  if context.output_hash["status"] != [ "deleted" ]
+  status = []
+  extract_status[record, status]
+  if status != [ "deleted" ]
     context.skip!
   end
 end
 
 to_field "id", extract_id
 to_field "record_update_date", extract_datestamp
-to_field "suppress_items_b", true
+to_field "suppress_items_b", Proc.new { |_, _| [true] }
```

Comment:
I think this needs push true to the accumulator
```suggestion
to_field "suppress_items_b", Proc.new { |_, acc| acc << true }
```

Reply:
@bibliotechy but the test case is working as expected.

Reply:
@dkinzer the test case doesn't test for the presence of this field

Reply:
Yup... 🤦